src/Box.js

```diff
@@ -6,7 +6,10 @@ import {
   width,
   height,
   maxWidth,
-  textAlign
+  textAlign,
+  border,
+  borderColor,
+  borderRadius
```

Comment:
I'm a little skeptical of adding `border`, `borderColor` and `borderRadius` to Box like this.

I've been trying to take the approach of "if we need this to be something that a user can change when implementing a feature using the component directly, then expose it via styled-system." In this current case, it feels like we're exposing these 3 props just so that we can set a border / border color and border radius on Flex which is used by the checkbox component.

I'm not opposed if we think this will be useful in the future, but I do want to be cognizant of guessing at future functionality. Is it worth this exposure here vs setting the border details on the checkbox component itself.

Reply:
Thinking this should be a part of a larger discussion where we evaluate both approaches, but I see it differently. I think all of styled-system's functions should be exposed and incrementally extended to all of our primitive components (Box, Flex, Grid, Text, Card). And those primitives should be used to build the rest of the components in Arbor.

Looking at styled-system's [table of props](https://github.com/jxnblk/styled-system/blob/master/docs/table.md), I'd expect `Box` to hold Core / Layout/ Misc / Position, `Text` to extend `Box` and hold Typography, `Flex` to extend `Box` and add Flexbox functions, etc.

Happy to have a conversation about it - we can't build an opinionated design system without consolidating several opinions :)